selfdrive/car/hyundai/values.py

```diff
@@ -1939,7 +1939,7 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
 
 # these cars require a special panda safety mode due to missing counters and checksums in the messages
 LEGACY_SAFETY_MODE_CAR = {CAR.HYUNDAI_GENESIS, CAR.IONIQ_EV_2020, CAR.IONIQ_EV_LTD, CAR.IONIQ_PHEV, CAR.IONIQ, CAR.KONA_EV,
-                          CAR.KIA_SORENTO, CAR.SONATA_LF, CAR.KIA_OPTIMA_G4, CAR.KIA_OPTIMA_G4_FL, CAR.VELOSTER,
```

Comment:
Make sure to add to unsupported long cars, since the radar disable doesn't work yer

Reply:
Right, thanks for calling that out  – [done](https://github.com/commaai/openpilot/pull/29502/commits/bbe123d1f9dd26a2ec7f94512c1629396a68b496).